ccd-definitions/src/main/java/uk/gov/hmcts/reform/divorce/ccd/event/DraftCreate.java

```diff
@@ -29,14 +29,10 @@ public void buildWith(final ConfigBuilder<CaseData, State, UserRole> builder) {
             .postState(DRAFT.getValue())
             .retries(120, 120)
             .grant("CRU", CITIZEN)
-            .grant("CRU", CASEWORKER_DIVORCE_COURTADMIN_BETA)
-            .grant("CRU", CASEWORKER_DIVORCE_COURTADMIN)
-            .grant("CRU", CASEWORKER_DIVORCE_SOLICITOR)
-            .grant("CRU", CASEWORKER_DIVORCE_SUPERUSER)
-            .grant("CRU", CASEWORKER_DIVORCE_COURTADMIN_LA)
-            .fields()
-            .optional(CaseData::getCreatedDate)
-            .optional(CaseData::getD8caseReference)
-            .optional(CaseData::getD8legalProcess);
+            .grant("R", CASEWORKER_DIVORCE_COURTADMIN_BETA)
```

Comment:
you can pass multiple roles into grant:

grant("R", ROLE_1, ROLE_2...)

Also if you find yourself using the same set of roles in multiple places you could store them in a collection and pass that in to grant.